examples/udp-echo.rs

```diff
@@ -236,15 +237,17 @@ fn extract_udp_packet(packet: &[u8]) -> Result<NaiveUdpPacket, wintun::Error> {
                     return Ok(udp_packet);
                 }
                 _ => {
-                    log::trace!("{:?} src={}, dst={}", protocol, src_addr, dst_addr);
+                    info = format!("{:?} src={}, dst={}", protocol, src_addr, dst_addr);
+                    log::trace!("{}", info);
```

Comment:
I would perfer to not log here and return the error directly. That way the caller can decide what to do with it and by default we remove non UDP logs which we are less likely to want in this example.

Reply:
OK